submissions/yhorodechna/side-menu-dom/js/script.js

```diff
@@ -55,88 +12,83 @@ function createNavList(obj) {
+                        container.querySelector('#' + activeElId).classList.remove('active')
                     }
                     activeElId = current.id;
                     current.classList.add('active')
-                    const item = items.find(f => f.id === current.id);
+                    const item = items.find(item => item.id === current.id);
                     navClick(current, item)
                 }
-                console.log(activeElId)
                 navEl.classList.remove('nav__full');
                 break;
             }
             else if (current.className && current.className.includes('header__link')) {
-                console.log('header link')
                 navEl.classList.add('nav__full');
                 break;
             }
             else if (current.className && current.className.includes('container')) {
-                console.log('end')
                 break;
             }
-            console.log('inside event listener', current)
             current = current.parentNode;
         }
-    })
-    parentEl.appendChild(div);
+    };
+    const navElements = items.map((item) =>
+        `<li class="nav__item">
+            <a id="${item.id}" onclick2="navLinkClick(this,event)" href="#" class="nav__link ${defaultId === item.id ? 'active' : ''}">
+                <span class="nav__text">${item.title}</span> 
+            </a>
+        </li>`
+    );
+    const html = `
+        <header class="header">
+            <a class="header__link" onclick2="headerLinkClick()" alt=""><img src="img/logo.png" alt="" class="img__btn"></a>
+            <h1 class="header__title">The Houses of Hogwarts</h1>
+        </header>
+        <nav id="nav" class="nav"><ul class="nav__list">${navElements.join('')}</ul></nav>
+                `;
+
+    const container = document.createElement("div");
+    container.className = 'container';
+    container.innerHTML = html;
+    container.addEventListener("click", containerClick)
+    parentEl.appendChild(container);
 };
 
-function navClick(current, navItem) {
-    console.log('linkClick', current, navItem)
-    const mainEl = document.querySelector('.main');
-    const item = DATA.find(f => f.id === current.id);
-    const wrapperEl = document.querySelector('.wrapper');
-    const titleEl = document.querySelector('.header__title');
+function navClick(current) {
+    showArticle(current.id);
+}
+
+function showArticle(id) {
+    const item = DATA.find(item => item.id === id);
+    const mainEl = document.querySelector('#main');
     const mainHtml = `
         <article class="main__article">
+                <h2 class="main__header">${item.title}</h2>
                 <img class="main__img" src="${item.img}" alt="">
                 <p class="main-description">${item.desc}</p>
                 <p class="main__quote">${item.quote} </p>
                 <p class="main__author">${item.author}</p>
```

Comment:
let's make this text semantic correct
https://developer.mozilla.org/ru/docs/Web/HTML/Element/blockquote

Reply:
Thanks! I didn't know about it. Solved